src/lib/metricsMgr/metricsMgr.h

```diff
@@ -1,3 +1,6 @@
+#ifndef SRC_LIB_METRICSMGR_METRICSMGR_H_
```

Comment:
Funny... github has "detected" this as a rename from commondebug_test.cpp to src/lib/metricsMgr/metricsMgr.h xD...

NTC